docs/connectors/kafka-connect-wrapper/overview.mdx

```diff
@@ -0,0 +1,158 @@
+---
+title: "Kafka Connect Wrapper"
+sidebar_label: "Overview"
+sidebar_position: 1
+---
+
+### General
+The Conduit Kafka Connect Wrapper provides the ability to use Kafka Connect plugins with Conduit. The connector can be used as a source or a destination but this depends on the Kafka Connect Connector being used. If the Kafka Connect Connector is a source connector, it should be used only as a source connector with this wrapper in Conduit.
```

Comment:
I may have missed a part of the conversation, but firstly we use "Kafka Connect plugins" here, and then Kafka Connect connector. Also, I _think_ connector should be lower cased.